lib/slack/web/api/templates/method.erb

```diff
@@ -44,6 +44,9 @@ module Slack
 <% if data['args']['user'] %>
             options = options.merge(user: users_id(options)['user']['id']) if options[:user]
 <% end %>
+<% if data['undocumented'] %>
+            logger.warn('<%= group %>.<%= name %> is an undocumented API endpoint.')
```

Comment:
Maybe "The <%= group %>.<%= name %> method is undocumented."

Reply:
👍